Documentation/getting-started.md

```diff
@@ -0,0 +1,128 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus
+```
+
+Next, create a ClusterRole that grants Prometheus the necessary permissions to discover and scrape the targets within the cluster.
+
+```yaml mdox-exec="cat example/rbac/prometheus/prometheus-cluster-role.yaml"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+- apiGroups: [""]
+  resources:
+  - nodes
+  - nodes/metrics
+  - services
+  - endpoints
+  - pods
+  verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources:
+  - configmaps
+  verbs: ["get"]
+- apiGroups:
+  - discovery.k8s.io
+  resources:
+  - endpointslices
+  verbs: ["get", "list", "watch"]
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingresses
+  verbs: ["get", "list", "watch"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+```
+
+Now, create a ClusterRoleBinding to bind the ClusterRole to the Prometheus ServiceAccount.
+
+```yaml mdox-exec="cat example/rbac/prometheus/prometheus-cluster-role-binding.yaml"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prometheus
+subjects:
+- kind: ServiceAccount
+  name: prometheus
+  namespace: default
+```
+
+Apply all these manifests to create the necessary RBAC resources. Now you are all set to deploy a Prometheus instance. Here is an example of a basic Prometheus instance manifest.
+
+```yaml mdox-exec="cat example/user-guides/getting-started/prometheus.yaml"
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: prometheus
+spec:
+  serviceAccountName: prometheus
+  resources:
+    requests:
+      memory: 400Mi
+  enableAdminAPI: false
+```
+
+To verify that the instance is up and running, run:
+
+```bash
+kubectl get -n default prometheus prometheus -w
+```
+
+For more information, see the [Prometheus Operator RBAC guide]({{< ref "rbac" >}}).
+
+## Deploying Alertmanager
+
+Let us take a simple example that creates 3 replicas of Alertmanager.
+
+```yaml mdox-exec="cat example/user-guides/alerting/alertmanager-example.yaml"
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  name: example
+spec:
+  replicas: 3
+```
+
+Wait for all Alertmanager pods to be ready:
+
+```bash
+kubectl get pods -l alertmanager=example -w
+```
+
+However, Alertmanager as it is now is of no use to us. To properly use Alertmanager, it is important to understand the relationship between Prometheus and Alertmanager. Alertmanager is used to:
+
+* Deduplicate alerts received from Prometheus.
+* Silence alerts.
+* Route and send grouped notifications to various integrations (PagerDuty, OpsGenie, mail, chat, …).
+
+So, to put Alertmanager instances to use, you would need to integrate it with Prometheus. Check out the next page on how to integrate Alertmanager with Prometheus.
```

Comment:
This page is quite small, what do you think about explaining the integration here as well?

Reply:
Ah ok, what I had in mind was mostly that Prometheus needs to know which Alertmanager he will send alerts to.

[Basically this part of the Prometheus Spec](https://prometheus-operator.dev/docs/api-reference/api/#monitoring.coreos.com/v1.AlertingSpec:~:text=the%20same%20shard.-,AlertingSpec,Alertmanager%20endpoints%20where%20Prometheus%20should%20send%20alerts%20to.,-AlertmanagerConfigMatcherStrategy)

Reply:
> [Basically this part of the Prometheus Spec](https://prometheus-operator.dev/docs/api-reference/api/#monitoring.coreos.com/v1.AlertingSpec:~:text=the%20same%20shard.-,AlertingSpec,Alertmanager%20endpoints%20where%20Prometheus%20should%20send%20alerts%20to.,-AlertmanagerConfigMatcherStrategy): Alertmanager endpoints where Prometheus should send alerts to

I have moved this part to the Getting-Started page along with [Exposing the Alertmanager Service](https://prometheus-operator.dev/docs/developer/alerting/#exposing-the-alertmanager-service). Now, need to think next about the [PrometheusRule](https://prometheus-operator.dev/docs/developer/alerting/#deploying-prometheus-rules) section. 

A separate page for just one section doesn't make sense. Maybe we can keep this section as it is in the Alerting page but with a different name. Wdyt?